test/cpp/microbenchmarks/bm_chttp2_hpack.cc

```diff
@@ -206,10 +257,24 @@ class RepresentativeClientInitialMetadata {
         GRPC_MDELEM_SCHEME_HTTP, GRPC_MDELEM_METHOD_POST,
         grpc_mdelem_from_slices(
             exec_ctx, GRPC_MDSTR_PATH,
-            grpc_slice_intern(grpc_slice_from_static_string("/foo/bar"))),
+            grpc_slice_from_static_string("/grpc.test.FooService/BarMethod")),
         grpc_mdelem_from_slices(exec_ctx, GRPC_MDSTR_AUTHORITY,
                                 grpc_slice_intern(grpc_slice_from_static_string(
                                     "foo.test.google.fr:1234"))),
+        grpc_mdelem_from_slices(
```

Comment:
Maybe break this out as a separate fixture also

Reply:
Restored the existing fixtures and made the new stuff new fixtures. PTAL